### PR TITLE
rb: --force flag forces bucket removal after BucketNotEmpty err

### PR DESCRIPTION
**Pull request by vadmeste**
Some users struggle to remove a bucket when they don't use recommanded
setups, such as NFS disks.

This command will force bucket removal in server side, using
x-minio-force-delete during DELETE bucket call.

### CODE REVIEW

File: cmd/client-fs.go [hunk 1]
```diff
@@ -894,6 +894,15 @@ func (f *fsClient) MakeBucket(ctx context.Context, region string, ignoreExisting
 	return nil
 }
 
+// RemoveBucket - remove a bucket
+func (f *fsClient) RemoveBucket(ctx context.Context, forceRemove bool) *probe.Error {
+	e := os.Remove(f.PathURL.Path)
```
Comment (klauspost) [inline cmd/client-fs.go:899]:
```suggestion
	e := os.RemoveAll(f.PathURL.Path)
```

**Review by klauspost — COMMENTED** [review]
LGTM. One suggestion.

File: cmd/client-fs.go (hunk 1 above)
Comment (vadmeste) [reply cmd/client-fs.go]: I made another change, PTAL.

**Review by harshavardhana — APPROVED** [review]

**Review by klauspost — APPROVED** [review]
LGTM

**Review by harshavardhana — APPROVED** [review]

**Pull request merged**